### DISCUSSION TITLE
New Member proposals and other requests

### DISCUSSION THREAD

**Discussion by cujomalainey** [thesofproject/sof#7383] [Team Posts]
As mentioned I have some requests for adding new members, here are the proposed additional members.

Virendra Pratap Arya
Github account: vp-arya
Associated email: vparya.dev@gmail.com

Carl Wakeland
Github account: cwakeland
Associated email: carl.wakeland@amd.com

Basavaraj Hiregoudar
Github account: bhiregou
Associated email: basavaraj.hiregoudar.1@gmail.com

Ben Zhang
Github account: bzhg
Associated email: benzh@google.com

Also sebcarlucci was wondering if it was fine if he sat in and listened to the next few TSCs while he is at Google so he could get some experience what open source TSC is like, any objections?

**Comment by dbaluta** [comment]
No objections. Curious if any of this people are coming from GSoC program.

Also, would be nice to understand what is their role and which area are they interested in.

**Comment by plbossart** [comment]
Np objections on having additional members if they can contribute to the project, but that would mean 15 members for a TSC, isn't that number starting to become too large too efficient/manageable?

**Comment by cujomalainey** [comment]
Top 3 are all AMD audio team. As far as I know no one has any relation to GSoC.

Ben is another chromeos audio engineer here at Google who has a tenure significantly longer than mine and has also been working with SOF, specifically with keyphrase detection code.

**Comment by cujomalainey** [comment]
@plbossart maybe we need an effective voting platform. I would think as other teams join and integrate then they are going to want to have a say on issues that define the platform.

**Comment by plbossart** [comment]
My remark was not really about voting and power. Voting should be rare and exceptional in technically-minded communities. consensus should be the assumed rule for "normal" development. 

it's more about efficiency. My experience with human teams is that above 8 contributors it becomes harder to have simple/direct conversations. It'd be fine with expanding the membership, but that would become more of a standardization workgroup and less of a technical board, which was my understanding initially. If we do have 15 or more people joining TSC, then we ought to think of a core team with representatives from each contributing companies. E.g. not everyone of those 15 people is going to go through issues and help prioritize resolution of specific problems.

**Comment by cujomalainey** [comment]
Understood, I would be ok with that.

**Comment by lgirdwood** [comment]
@cujomalainey I've added all to SOF project, are all to be also added to TSC ?

**Comment by lgirdwood** [comment]
Fwiw, we could draft some constitution to say have 2 from each company if we find the SNR on the call is not good. But lets see how things go, not everyone from each company attends all the time anyway.

**Comment by cujomalainey** [comment]
Ideally yes it would be good if we can get them involved.

**Comment by lgirdwood** [comment]
@vp-arya @cwakeland @bhiregou @bzhg welcome to the TSC !
